TypeCobol.LanguageServer/Workspace.cs

```diff
@@ -56,6 +59,13 @@ public class Workspace
         public Queue<MessageActionWrapper> MessagesActionsQueue { get; private set; }
         private Func<string, Uri, bool> _Logger;
 
+#if EUROINFO_RULES
+        /// <summary>
+        /// The Instance of the Cpy Copy names Map
+        /// </summary>
+        public CopyNameMapFile CpyCopyNamesMap { get; set; }
```

Comment:
Workspace does not need to keep an instance of the copy name map. I think TypeCobolOptions should hold the value of the path to the file and CompilationProject should hold the instance of the loaded file.
- path is a new option so it works just like all other options
- CompilationProject is responsible for loading the file (either at construction time or lazily at first usage of the HasCpyCopy method)

Reply:
This is an option,  at option resolution time it should be determinated if the file can be loaded, it shall not be a compilation error.